tests/trainer/test_trainer.py

```diff
@@ -1441,3 +1448,61 @@ def test_trainer_profiler_incorrect_arg_type(profiler):
                        match=r"Only None, bool, str and subclasses of `BaseProfiler`"
                              r" are valid values for `Trainer`'s `profiler` parameter. *"):
         Trainer(profiler=profiler)
+
+
+def _get_pytorch_profiler_total_duration(events):
+    total_time = sum([e.cpu_time + e.cuda_time for e in events])
+    return total_time / 1e6  # convert microseconds to seconds
+
+
```

Comment:
would be nice to have a test for the nested scopes too (just to make sure they don't fail)

Reply:
Done !